luna/featurevis/images.py

```diff
@@ -116,6 +116,8 @@ def initialize_image_ref(
         image_f = fft_image(img, std=std)
     else:
         std = std or 0.01
+        # For replication purposes
+        np.random.seed(1)
```

Comment:
Do we want this in the framework? Or should it be an option one can set?

Reply:
This just impacts on the random image initialization and it does not effect the outcome much. Lucid had also some fixed randomization for generalizing the same result. Though, if you still don't think it should be here then let's keep it as an option for testing purposes.

Reply:
Maybe we can do:

```
def initialize_image_ref(
    width, height, std=None, fft=True, decorrelate=True, channels=None, seed=None
):
    """
    Creates an initial randomized image to start feature vis process.
    This could be subject to optimization in the future.
    Args:
        width (int): The width of the image.
        height (int): The height of the image.
        sd (float): standard deviation for noise initialization.
        fft (boolean): Image parameterization with fast fourier transformation.
        deccorelate (boolean): the color interpretation of the image tensor's color.
        channels (boolean): True for gray images.
        seed (int): Random seed for the image initialization for reproducibility.
    Returns:
        A randomly generated image.
    """
    print("initializing image")
    # We start from a gray image with some random noise
    if tf.compat.v1.keras.backend.image_data_format() == "channels_first":
        img = tf.random.uniform((1, 3, width, height), dtype=tf.dtypes.float32)
    else:
        img = tf.random.uniform((1, width, height, 3), dtype=tf.dtypes.float32)
    if fft:
         image_f = fft_image(img, std=std)
     else:
         std = std or 0.01
         if !(seed is None):
             np.random.seed(seed)
```

Reply:
Agreed!